packages/platform-android/src/commands/runAndroid/index.ts

```diff
@@ -356,7 +356,7 @@ function startServerInNewWindow(
       encoding: 'utf8',
       flag: 'w',
     });
-    return execa.sync('cmd.exe', ['/C', launchPackagerScript], {
+    return execa('cmd.exe', ['/C', launchPackagerScript], {
```

Comment:
this is now async, so technically we should make the `startServerInNewWindow` async and await it inside `isPackagerRunning`

Reply:
Will the other platforms run `execa` command directly?

Reply:
I haven't encounter issues with `execa.sync` on other platforms (at least macOS), so it's likely Windows-only quirk.

Reply:
I think if `execa.sync` works on other platforms, `execa` directly _should_ work too.

Reply:
Feel free to convert other cases in this function to async, I don't mind :). I avoided doing so, because I didn't want to propagate `async` all the way up, but since it's now necessary, there's no excuse anymore :)

Reply:
Weird happening. Using async/await brings back this issue again. Can we revert back to `spawn` for Windows only or just use `execa` without async/await?

Reply:
Let's go with `execa` then. But add a comment that it's not awaited on purpose, because the script on Windows doesn't finish or something

Reply:
Done.